plugins/value-container-assignment/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+description = "Kotlin Value Container Assignment Compiler Plugin"
+
+plugins {
+    kotlin("jvm")
+    id("jps-compatible")
+}
+
+dependencies {
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.common"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.k1"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.k2"))
+    embedded(project(":kotlin-value-container-assignment-compiler-plugin.cli"))
+
+    testApi(project(":compiler:backend"))
+    testApi(project(":compiler:cli"))
+    testApi(project(":kotlin-value-container-assignment-compiler-plugin.cli"))
+    testCompileOnly(project(":kotlin-compiler"))
+    testImplementation(project(":kotlin-scripting-jvm-host-unshaded"))
+
+    testApiJUnit5(vintageEngine = true)
```

Comment:
Why do you enable vintage engine if all tests are using only new test infrastructure with JUnit5?

Reply:
Copy/paste bug. Removed